setup/l10n_it_vat_payability/setup.py

```diff
@@ -2,5 +2,12 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
+    odoo_addon={
+        'external_dependencies_override': {
+            'python': {
+                'openupgradelib': 'git+https://github.com/OCA/openupgradelib.git@master',
```

Comment:
These overrides end up in the `install_requires` section of setup.py. And these must be PEP 508 compliant. So the proper syntax would be `openupgradelib @ git+https://github.com/OCA/openupgradelib.git@master`.

But specifying an URL here is not ideal, because it basically obliges all downstream users to use that exact same URL in their project. They could not pin a specific commit for instance. 

So can you try removing this `external_dependencies_override` here, and adding `openupgradelib @ git+https://github.com/OCA/openupgradelib.git@master` in `test-requirements.txt`.

Let me know if that does not work.

Reply:
This is now working, thanks!